app-ios/Modules/Sources/Contributor/ContributorView.swift

```diff
@@ -1,18 +1,25 @@
 import Component
-import KMPContainer
+import Dependencies
 import Model
 import shared
 import SwiftUI
 
 public struct ContributorView: View {
     @State var presentingURL: IdentifiableURL?
-    @ObservedObject var viewModel: ContributorViewModel = .init()
 
     public init() {}
 
     public var body: some View {
+        Group {
+//            ContributorSwiftUIView { url in
+//                presentingURL = .init(string: url)
+//            }
+
+            ContributorComposeView { url in
+                presentingURL = .init(string: url)
+            }
+        }
         ContributorComposeView(
```

Comment:
@ry-itto 
Is it unnecessary now? 👀

Reply:
Removed https://github.com/DroidKaigi/conference-app-2023/pull/1054/commits/ec4894414ccce15c7bb3c465bd7903a743e9438c